client/src/components/TeacherService.ts

```diff
@@ -0,0 +1,12 @@
+export const getRooms = () => {
+    return [
+      { id: 1, roomNumber: 'A100' },
+      { id: 2, roomNumber: 'A102' },
+      { id: 3, roomNumber: 'A103' },
+      { id: 4, roomNumber: 'A201' },
+      { id: 5, roomNumber: 'A310' },
+      { id: 6, roomNumber: 'A202' },
+      { id: 7, roomNumber: 'A107' },
+    ];
+    }
```

Comment:
отформатируй код, убери лишний return и добавь тип возвращаемого значения
```suggestion
export const getRooms = () => [
      { id: 1, roomNumber: 'A100' },
      { id: 2, roomNumber: 'A102' },
      { id: 3, roomNumber: 'A103' },
      { id: 4, roomNumber: 'A201' },
      { id: 5, roomNumber: 'A310' },
      { id: 6, roomNumber: 'A202' },
      { id: 7, roomNumber: 'A107' },
    ];
```